pkg/query/storeset.go

```diff
@@ -85,6 +85,7 @@ type StoreSet struct {
 	storesStatusesMtx     sync.RWMutex
 	stores                map[string]*storeRef
 	storeNodeConnections  prometheus.Gauge
+	storeNodeStatus       *prometheus.GaugeVec
```

Comment:
Can we change the name?

Node sounds again like related to `host` to me.

Proposition:
```suggestion
	storeUpnessMetric       *prometheus.GaugeVec
```